src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -365,8 +365,8 @@ export class ConnectionWidget extends lifecycle.Disposable {
 
 		if (this._azureAccountDropdown) {
 			this._register(styler.attachSelectBoxStyler(this._azureAccountDropdown, this._themeService));
-			this._register(this._azureAccountDropdown.onDidSelect(() => {
-				this.onAzureAccountSelected();
+			this._register(this._azureAccountDropdown.onDidSelect(async () => {
```

Comment:
You don't need to async await this, just return the promise

Reply:
Actually this shouldn't even be awaited, since the caller won't care about the promise bring generated you should just catch the error